dao/src/main/java/greencity/entity/user/ubs/Address.java

```diff
@@ -69,4 +69,16 @@
 
     @Embedded
     private Coordinates coordinates;
+
+    @Column(name = "city_en", nullable = false)
```

Comment:
Is it really need to declare in annotation ?

Reply:
no need as the filed name is the same